libkbfs/file_data.go

```diff
@@ -321,7 +162,7 @@ func (fd *fileData) getByteSlicesInOffsetRange(ctx context.Context,
 				firstBlockOff = iptr.Off
 			}
 			if i == len(pfr)-1 {
-				leafBlock := blockMap[iptr.BlockPointer]
+				leafBlock := blockMap[iptr.BlockPointer].(*FileBlock)
```

Comment:
Did you mean to do type assertion without the `ok` parameter here? Seems like this can be dir block in the future too.

Reply:
This will always only be in `*FileBlock`, because this function deals with byte slices, and that's not relevant for directories.